.github/workflows/test.yml

```diff
@@ -138,6 +138,90 @@ jobs:
           - name: npm install
             run: npm install
           - name: run tests
-            run: npm run test
+            run: npm run test:unit
       env:
           CI: true
+
+  frontend-e2e-tests:
+    runs-on: ubuntu-latest
+    name: Front-end E2E tests
+    strategy:
+      matrix:
+        php-version: [ '8.1' ]
+        nextcloud-version: [ 'master' ]
```

Comment:
would it make sense to test stable* too?

Reply:
I added the oldest and most recent supported versions. As this repo is not seeing a lot of activity and there are only 2 test cases it should be fine.